kong/plugins/opentelemetry/otlp.lua

```diff
@@ -72,9 +77,39 @@ local function transform_events(events)
   return pb_events
 end
 
+-- this function is to translate the span to otlp span, but of universal span format
```

Comment:
I don't understand what "but of universal span format" means.

Reply:
The original design to support arbitrary types of tracing plugins has not made it clear:
1. We have an internal representation for span, and it should preserve what it received from downstream requests at best effort;
2. when exporting the data to collectors, it needs to transform the span to meet the requirement with its field (DD, OTEL, etc), and then translate it into a format used for the reporting API (gRPC, JSONRPC, etc). This function does the first step but not the second one.